<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircleCreation</class>
 <widget class="QMainWindow" name="CircleCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBox_centerX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_centerY">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_centerBrackets">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(          ,          )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_center">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Центр окружности</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ringBrackets">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(          ,          )</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_ringX">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_ringY">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_ring">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Точка на окружности</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>215</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_randomise">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_randomMin">
       <item>
        <widget class="QLabel" name="label_randomMin">
         <property name="text">
          <string>Нижняя граница случайных значений</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_randomMin">
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_randomise">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="recources.qrc">
         <normaloff>:/D:/Users/user/Downloads/dice.png</normaloff>:/D:/Users/user/Downloads/dice.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_randomMax">
       <item>
        <widget class="QLabel" name="label_randomMax">
         <property name="text">
          <string>Верхняя граница случайных значений</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_randomMax">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
